src/components/views/elements/Confetti.js

```diff
@@ -0,0 +1,199 @@
+const confetti = {
+    //set max confetti count
+    maxCount: 150,
+    //syarn addet the particle animation speed
+    speed: 3,
+    //the confetti animation frame interval in milliseconds
+    frameInterval: 15,
+    //the alpha opacity of the confetti (between 0 and 1, where 1 is opaque and 0 is invisible)
+    alpha: 1.0,
+    //call to start confetti animation (with optional timeout in milliseconds)
+    start: null,
+    //call to stop adding confetti
+    stop: null,
+    //call to stop the confetti animation and remove all confetti immediately
+    remove: null,
+    isRunning: null,
+    //call and returns true or false depending on whether the animation is running
+    animate: null,
+};
+
+(function() {
+    confetti.start = startConfetti;
+    confetti.stop = stopConfetti;
+    confetti.remove = removeConfetti;
+    confetti.isRunning = isConfettiRunning;
+    confetti.animate = animateConfetti;
```

Comment:
this is a good indicator that this should be a real class rather than a functional type.